env.sample

```diff
@@ -2,6 +2,7 @@ export API_HOST=localhost
 export PORT=2015
 export POSTGRE_CONNECTION_STRING=postgresql://localhost:5432/webmaker
 export API_VERSION=dev
+export ID_SERVER_CONNECTION_STRING=http://localhost:6767
```

Comment:
Can we default this to the staging instance of `id.webmaker.org` rather than requiring the user to spin up a local instance?